.cargo/config_fast_builds.toml

```diff
@@ -1,40 +1,105 @@
-# Add the contents of this file to `config.toml` to enable "fast build" configuration. Please read the notes below.
-
-# NOTE: For maximum performance, build using a nightly compiler
-# If you are using rust stable, remove the "-Zshare-generics=y" below.
+# Copy this file to `config.toml` to speed up your builds.
+#
+# This configuration enables the following:
+#
+# - Faster linker (stable)
+#   - LLD is a faster drop-in replacement for the default linker, part of the LLVM project.
+#     It requires installation, please see the relevant section of this config for your platform.
+#   - Mold / Sold is a linker faster than even LLD, but with limited platform support.
+#     It requires installation, please see the relevant section of this config for your platform.
+#     Mold is precompiled for Linux, but must be manually built on MacOS and is unavailable for Windows.
+#   - Please note that LLD and Mold are incompatible with one another. Pick one and comment out the other!
+# - Nightly (unstable) features
+#   - `share-generics` prevents duplicating monomorphized generic code across crates.
+#   - `threads` makes use of the parallel frontend to increase compiler performance.
```

Comment:
This makes it seem like the config enables mold and the nightly features too, while that's not the case as they are commented out. I would make it more clear here that the user will have to uncomment the options that they want to enable.

Reply:
I've greatly improved this section in 49999bbeb6e0b0574624796ff4ab7011a9388411, would you mind taking a look?

Reply:
Honestly I think this is still a bit too far down. People **love** to skip lengthy explanations, I can imagine someone reading "Copy this file to `config.toml` to speed up your builds" and then thinking that's it and the rest is just optional. I would put a brief mention of the fact they have to uncomment the relevant lines and potentially also install something right in the first line.